docker/Dockerfile.1-13-gpu

```diff
@@ -0,0 +1,65 @@
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION}-gpu as tfs
+FROM nvidia/cuda:10.0-base-ubuntu16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+# https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/tools/docker/Dockerfile.gpu
+ENV CUDNN_VERSION=7.4.1.5
+ENV TF_TENSORRT_VERSION=5.0.2
+
+RUN \
+    apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        cuda-command-line-tools-10-0 \
+        cuda-command-line-tools-10-0 \
```

Comment:
whats difference wrt previous line ?

Reply:
This is copied from the official Tensorflow serving repo. I have noticed this too. will fix